### PR TITLE
Vet clinic database: add "join table" for visits

### PR DESCRIPTION
**Pull request by amejid**
## Milestone 4: Add "join table" for visits

In this PR, the following were implemented:-

- Create many-to-many relationships

The queries look like the following:-

- Who was the last animal seen by William Tatcher?
q1
- How many different animals did Stephanie Mendez see?
q2
- List all vets and their specialties, including vets with no specialties.
q3
- List all animals that visited Stephanie Mendez between April 1st and August 30th, 2020.
q4
- What animal has the most visits to vets?
q5
- Who was Maisy Smith's first visit?
q6
- Details for most recent visit: animal information, vet information, and date of visit.
q7
- How many visits were with a vet that did not specialize in that animal's species?
q8
- What specialty should Maisy Smith consider getting? Look for the species she gets the most.
q9

### CODE REVIEW

File: queries.sql [hunk 1]
```diff
@@ -60,4 +60,80 @@ INNER JOIN animals
...
+
+SELECT species.name AS species_name, COUNT(*)  FROM visits
+INNER JOIN vets
+ON visits.vet_id = vets.id
+INNER JOIN animals
+ON visits.animal_id = animals.id
+INNER JOIN species
+ON animals.species_id = species.id
+WHERE vets.name = 'Maisy Smith'
+GROUP BY species.name
```
Comment (Whoistolu) [inline queries.sql]: - [ ] [REQUIRED] Great job @amejid , however, it looks like you skipped some parts of the project requirements that say you should create queries to answer the following questions in transactions:

- Who was the last animal seen by William Tatcher?
Kindly implement this to get the project approved.

**Review by Whoistolu — CHANGES_REQUESTED** [review]
Hi @amejid 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

## Required Changes ♻️

- [ ]  _Check the inline comments under the review for sections to improve._

### Optional suggestions
Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better.

### N/A

Cheers and Happy coding!

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me @whoistolu in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by rustam-tolipov — APPROVED** [review]
## Hello @amejid 👋, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
## Congratulations! 🎉 🎉 🎉 🎉 🎉 

Cheers and Happy coding!👏👏👏

Developer-512px-10

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._
```

**Comment by amejid** [comment]
Thank you for reviewing and approving my code.

File: queries.sql (hunk 1 above)
Comment (amejid) [reply queries.sql]: Thank you for reviewing my project, I will make the necessary corrections as suggested.

**Pull request merged**